src/App.tsx

```diff
@@ -1,35 +1,54 @@
 import React from 'react';
 import './App.scss';
+
 import { MoviesList } from './components/MoviesList';
+import { Search } from './components/Search';
+
 import moviesFromServer from './api/movies.json';
 
-type State = {};
+type State = {
+  query: string;
+  movies: Movie[];
+};
 
 export class App extends React.Component<{}, State> {
-  state: State = {};
+  state: State = {
+    query: '',
+    movies: moviesFromServer,
+  };
+
+  getQuery = (query: string) => {
+    this.setState({ query });
+  };
+
+  changeVisibleMovies = () => {
```

Comment:
ты не меняешь, ты получаешь
```suggestion
  getVisibleMovies = () => {
```